amd/src/board.js

```diff
@@ -319,4 +335,35 @@ export default class extends KanbanComponent {
     hideDropZone() {
         this.getElement(selectors.ADDCOLUMNCONTAINER).classList.remove('mod_kanban_insert');
     }
+
+    /**
+     * Scroll to the left.
+     */
+    _scrollLeft() {
+        this.getElement(selectors.MAIN).scrollLeft -= 80;
+    }
+
+    /**
+     * Scroll to the right.
+     */
+    _scrollRight() {
+        this.getElement(selectors.MAIN).scrollLeft += 80;
+    }
+
+    /**
+     * Only show scroll buttons if it's possible to scroll in this direction.
+     */
+    _updateScrollButtons() {
+        let main = this.getElement(selectors.MAIN);
+        if (main.scrollLeft <= 1) {
+            this.getElement(selectors.SCROLLLEFT).style.setProperty('visibility', 'hidden');
+        } else {
+            this.getElement(selectors.SCROLLLEFT).style.setProperty('visibility', 'visible');
+        }
+        if (main.scrollLeft < main.scrollLeftMax) {
```

Comment:
Not a standard property, does not work on Chrome and many other browsers, see: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeftMax?retiredLocale=de

Needs to be changed.